x-pack/plugins/threat_intelligence/kibana.json

```diff
@@ -14,7 +14,8 @@
     "unifiedSearch",
     "kibanaUtils",
     "navigation",
-    "kibanaReact"
+    "kibanaReact",
+    "timelines"
```

Comment:
is it needed when padding the timeslines service via context?

Reply:
nope it's not, thanks for catching this! This was a left over of an earlier attempt to integrate timeline :)